core/src/main/java/org/apache/accumulo/core/client/admin/CloneConfiguration.java

```diff
@@ -98,21 +98,29 @@ public static interface Builder {
      * @param keepOffline
      *          true if the new table is to be kept offline after cloning.
      */
-    public Builder setKeepOffline(boolean keepOffline);
+    Builder setKeepOffline(boolean keepOffline);
 
     /**
      * Build the clone configuration
      *
      * @return the built immutable clone configuration
      */
-    public CloneConfiguration build();
+    CloneConfiguration build();
   }
 
   /**
    * @return a {@link CloneConfiguration} builder
    */
-  public static CloneConfiguration.Builder builder() {
+  static CloneConfiguration.Builder builder() {
     return new CloneConfigurationImpl();
   }
 
+  /**
+   * @return an empty configuration object with the default settings.
+   * @since 2.1.0
+   */
+  static CloneConfiguration empty() {
```

Comment:
I wanted to keep it as short as possible since it's likely to be used as a param to the clone method.
<pre>
client.tableOperations().clone(table1, table2, CloneConfiguration.empty()));
</pre>
I also thought `none()` or `default()` might work. Or we could even just have a static `build()` but I thought that might be confusing with the `builder()` method.

Reply:
Understood. Makes sense to me :+1: